docs/extend/_shared/manifest-extension-runtime.md

```diff
@@ -1,8 +1,8 @@
 
 | Property | Description | Notes |
 | --- | --- | --- |
-| **scopes** | *An array of authorization scopes (strings) listing permissions required by your extension.* | For example, `vso.work` and `vs.code_write` indicates your extension needs read-only access to work items and read/write access to source code (and related resource). Scopes are presented to the user when installing your extension. |
-| **demands** | *An array of demands (strings) listing the capabilities required by your extension.* | For example, `api-version/3.0` indicates your extension uses version 3.0 APIs and therefore cannot run in older products that do not support this version. |
+| **scopes** | *An array of authorization scopes (strings) listing permissions required by your extension.* | For example, `vso.work` and `vs.code_write` indicates your extension needs read-only access to work items and read/write access to source code (and related resource). Scopes are presented to the user when installing your extension. See all available scopes [here](../develop/manifest.md#scopes).|
```

Comment:
Link should probably be around "all available scopes". "Here" links are bad for screen reader users since they just get a page full of "here" links when using a link view in screen readers